src/status_im2/contexts/wallet/account/tabs/view.cljs

```diff
@@ -1,13 +1,57 @@
 (ns status-im2.contexts.wallet.account.tabs.view
   (:require
     [quo2.core :as quo]
+    [quo2.foundations.colors :as colors]
     [react-native.core :as rn]
     [status-im2.contexts.wallet.account.style :as style]
```

Comment:
why are we using the style file from  `status-im2.contexts.wallet.account.style`. imo we should only use one style per view file and if we need something reused then we make a component from it 👍

Reply:
also on the topic of namespaces perhaps it's better we split these into separate namespaces dependent on topic
e.g
`status-im2.contexts.wallet.account.tabs.view` - main entry point requiring the following sub-view
`status-im2.contexts.wallet.account.tabs.about.view` (or status-im2.contexts.wallet.account.tabs.about)
`status-im2.contexts.wallet.account.tabs.assets.view` (or status-im2.contexts.wallet.account.tabs.assets)
`status-im2.contexts.wallet.account.tabs.collectibles.view`  (or status-im2.contexts.wallet.account.tabs.collectibles)
etc 
it seems like this file could be quite huge otherwise

Reply:
@J-Son89 that's right, splitted it into a different namespace

Reply:
@OmarBasem I see this file is still using the styles from `[status-im2.contexts.wallet.account.style :as style]`. Imo this file should be using styles from "src/status_im2/contexts/wallet/account/tabs/style.cljs" - we should have one style file per view file 👍 even if it's a bit of copy pasta

Reply:
Alright will update it 👍